extensions-core/druid-ranger-security/pom.xml

```diff
@@ -178,6 +178,10 @@
                     <groupId>org.apache.hadoop</groupId>
                     <artifactId>hadoop-common</artifactId>
                 </exclusion>
+                <exclusion>
+                    <groupId>com.amazonaws</groupId>
+                    <artifactId>aws-java-sdk-bundle</artifactId>
+                </exclusion>
```

Comment:
it'd be nice to add some comments to explain how those are safe to exclude in our case and why we are excluding them.

Reply:
AWS-sdk-bundle is used only in CloudWatch audit [destination](https://github.com/apache/ranger/blob/master/agents-audit/src/main/java/org/apache/ranger/audit/destination/AmazonCloudWatchAuditDestination.java), and I believe that codepath is not touched by ranger extension for Druid, 
I will add  [aws-java-sdk-logs](https://mvnrepository.com/artifact/com.amazonaws/aws-java-sdk-logs/1.12.669) package as explicit dependency of this module.